tasks/configure.yml

```diff
@@ -22,6 +22,9 @@
   notify: ["mongodb restart"]
   when: mongodb_transparent_hugepage_disable
 
+- name: mongodb start
+  service: name=mongod state=started daemon-reload=true
```

Comment:
rewrite following the "hash" syntax:

service: 
  name: mongod
  state: started
  daemon-reload: true